ontopy/ontology.py

```diff
@@ -283,39 +283,45 @@ def get_by_label(
         The current implementation also supports "*" as a wildcard
         matching any number of characters. This may change in the future.
         """
-        # pylint: disable=too-many-arguments,too-many-branches
+        # pylint: disable=too-many-arguments,too-many-branches,invalid-name
         if not isinstance(label, str):
             raise TypeError(
                 f"Invalid label definition, must be a string: {label!r}"
             )
+
+        # Comment to reviewer:
+        # Do we really want to disallow space in label?
+        # I think we wanted ontopy to work with any ontology, also ontologies
+        # having space in their labels.
+        # Rather than enforcing EMMO conventions here, I think the user should
+        # run emmocheck on the ontology - it will alert about labels with
+        # spaces
```

Comment:
I agree, please add this as an issue.

Reply:
Issue added. Ok to fix and close it in this PR?
We just have to remove the next 4 lines.

Reply:
OK